### DISCUSSION TITLE
Welcome to Solnet Discussions!

### DISCUSSION THREAD

**Discussion by hoakbuilds** [bmresearch/Solnet#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chilitechno** [comment]
Did this project die? No releases since July...

**Comment by ghost** [comment]
I'm wondering, too... It would be a pity, since Solnet is a really good approach. Does anyone know how to deal with Solana using C# instead?
